packages/flutter/test/widgets/safe_area_test.dart

```diff
@@ -184,6 +218,44 @@ void main() {
         final Offset finalPoint = tester.getCenter(find.byType(Placeholder));
         expect(initialPoint, finalPoint);
       });
+
+      testWidgets(
+          'SafeArea with nested Scaffold  - partial ViewInsets consume Padding',
+          (WidgetTester tester) async {
```

Comment:
Style guide nit: These should all be one line

Reply:
Fixed in commit c82919c